share/tools/web_config/sample_prompts/nim.fish

```diff
@@ -24,6 +24,12 @@ function fish_prompt
     # │ 1	15048	0%	arrêtée	sleep 100000
     # ╰─>$ echo there
 
+    if test $status = 0
```

Comment:
You'll want to define `retc` as a local variable outside of this block:

```fish
set -l retc
if test $status = 0
```

and once you're doing that, you can just set it to red directly:

```fish
set -l retc red
test $status = 0; and set retc green
```

Reply:
done